dev_requirements.txt

```
@@ -30,7 +31,7 @@ nbsphinx==0.9.1
 matplotlib==3.8.1
 pandas==2.1.3
 pyscf==2.5.0; sys_platform != 'win32'
-openfermion==1.6.0; sys_platform != 'win32'
+-e git://github.com/quantumlib/OpenFermion.git
```

Great news, OpenFermion patch released!
```suggestion
openfermion==1.6.1; sys_platform != 'win32'
```